libs/platform/form/src/renderers/default-form.renderer.ts

```diff
@@ -208,6 +208,7 @@ export class DefaultFormRenderer implements FormRenderer {
         floatLabel=${ifDefined(field.floatLabel)}
         .style=${this.resolveStyles(field)}
         ?hasError=${field.attributes?.hasError}
+        ?required=${field.required}
```

Comment:
We have to add it only for text field?

Reply:
There are a few components for which this does not apply like toggle, and doesn't show up even when hydrated.